src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -368,6 +369,12 @@ function unbound_configure_do($verbose = false, $unused = '')
         }
     }
 
+    if (isset($config['unbound']['stats'])) {
+        @touch('/var/unbound/data/stats');
```

Comment:
this is for deciding to start the logger in start.sh?

Reply:
correct, but also to determine if logging should take place from the perspective of unbound